packages/react/src/wallet/hooks/useInstalledWallets.ts

```diff
@@ -3,18 +3,20 @@ import { assertWindowEthereum } from "@thirdweb-dev/wallets";
 export function useInstalledWallets() {
   let isMetamaskInstalled = false;
   let isCoinbaseWalletInstalled = false;
+  let isZerionWalletInstalled = false;
 
   if (assertWindowEthereum(globalThis.window)) {
     isMetamaskInstalled = globalThis.window.ethereum?.isMetaMask;
     isCoinbaseWalletInstalled =
       globalThis.window.ethereum?.isCoinbaseWallet ||
-      globalThis.window.ethereum?.providers?.some((p) => p.isCoinbaseWallet) ||
-      false;
```

Comment:
Array.prototype.some always returns boolean. Therefore `|| false` is not necessary.
Since Array is an optional chaining, it may return `undefined`, but since everything uses `double not` before return, it will be a `Boolean` when used by the user.

Reply:
@joaquim-verges fixed it.
https://github.com/thirdweb-dev/js/pull/1210/commits/dd1583ea6cb0cb298f7671abdc8b34db94d41afe

Reply:
can we do `!!` instead of `Boolean()` here? (just convention)

Reply:
@konojunya any chance you could switch to `!!` instead of `Boolean` (and fix the conflicts)? thank you!